src/lib.rs

```diff
@@ -301,6 +353,17 @@ impl<T> ArcMutexGuardian<T> {
     pub fn take(handle: sync::Arc<sync::Mutex<T>>) -> sync::LockResult<ArcMutexGuardian<T>> {
         take!(handle, sync::MutexGuard<'static, T>, ArcMutexGuardian, lock)
     }
+
+    pub fn try_take(
```

Comment:
Docs please — I'd recommend taking from [the standard library](https://doc.rust-lang.org/1.78.0/src/std/sync/mutex.rs.html#322-340). Ditto for the other `try_take` methods.